webf/lib/src/devtools/modules/css.dart

```diff
@@ -75,6 +74,30 @@ class InspectCSSModule extends UIInspectorModule {
     }
   }
 
+  Map<String, dynamic> parseAnnotatedStyle(String text) {
+    Map<String, String> styles = {};
```

Comment:
Why not use CSSParser instead of write your own?

Reply:
function(parseAnnotatedStyle) have two purpose
1、parse should disabled property
2、get a clean style text ,without annotation
CSSParser used to be parse style property from a text，I don't know which property should be removed；

Reply:
fixed bugs;